build/AzurePipelinesTemplates/WindowsAppSDK-BuildVSIX-Steps.yml

```diff
@@ -0,0 +1,236 @@
+parameters:
+- name: SignOutput
+  type: boolean
+  default: False
```

Comment:
If this yml file is only being called from build/AzurePipelinesTemplates/WindowsAppSDK-BuildVSIX-Stage.yml, then at least currently, this parameter can only remain False? I ask because if that is expected to remain the case at least in the short term, then we don't really need to port WindowsAppSDK-SignPackageContents-steps.yml over?

Reply:
Oh yea I should have made this clear in the description. We are removing it from the Aggregator repo.